lib/screens/explore_screens/vote_screens/vote/interactor/viewmodels/proposal_type_model.dart

```diff
@@ -44,28 +44,16 @@ const List<ProposalType> proposalTypes = [
     index: 1,
     type: 'Upcoming',
     proposalStage: 'staged',
-    referendumStage: 'staged',
+    referendumStage: ['staged'],
     indexPosition: 3,
     isReverse: false,
   ),
   ProposalType(
     index: 2,
     type: 'History',
     proposalStage: 'done',
-    referendumStage: 'passed',
+    referendumStage: ['passed', 'failed'],
     indexPosition: 3,
     isReverse: true,
   ),
 ];
-
-// Proposals
-// 'Open': {'stage': 'active', 'status': 'open'},
-// 'Evaluate': {'stage': 'active', 'status': 'evaluate', 'reverse': 'true'},
-// 'Passed': {'stage': 'done', 'status': 'passed', 'reverse': 'true'},
-// 'Failed': {'stage': 'done', 'status': 'rejected', 'reverse': 'true'},
```

Comment:
@n13 There is no way to know from a status field as in the proposals, if a referendum is `passed` or `failed`?
or
Just because it is the result to a call passed or failed?